packages/components-css/skip-link-css/src/_mixin.scss

```diff
@@ -1,3 +1,18 @@
-/* stylelint-disable-next-line block-no-empty */
 @mixin nl-skip-link {
+  background-color: var(--nl-skip-link-background-color, Canvas);
+  color: var(--nl-skip-link-color, LinkText);
+  inset-inline-start: -100%;
+  min-block-size: var(--nl-skip-link-min-block-size, 44px);
+  min-inline-size: var(--nl-skip-link-min-inline-size, 44px);
+  padding-block: var(--nl-skip-link-padding-block);
+  padding-inline: var(--nl-skip-link-padding-inline);
+  position: fixed;
+  text-decoration: var(--nl-skip-link-text-decoration);
+}
```

Comment:
Misschien moeten we een `nl-skip-link--top` hebben die 'm op `display: none; position: fixed;` zet? (een verwijzing naar `#top`)

Reply:
@Robbert dit begrijp ik niet goed.
Wil je de nl-skip-link op display none zetten als deze onzichtbaar is?

Reply:
Dat klopte niet inderdaad. Ik bedoelde:

Misschien moeten we een `nl-skip-link--top` hebben (verwijzing naar `#top`) die 'm op zichtbaar maakt met `position: fixed`. Zonder `--top` is 'ie `position: static`, dan zou je 'm eventueel ook op andere plekken als skip link kunnen inzetten.

Reply:
Ik vraag me af of dit onder de categorie skip link valt, dit lijkt me meer een aparte component.
Maar dan zou ik un niet display:none maken omdat de link dan uit de link-lijst weg is voor screen readers en misschien wel handig.

Reply:
Dit omdat deze link vaak een ander gedag vertoont: is meestal altijd zichtbaar en heeft vaak zo’n leuk smooth scroll effect.